backend/prometheus_app/core_services_monitoring.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI, Response
+from prometheus_client import Gauge, CollectorRegistry, generate_latest, CONTENT_TYPE_LATEST
+import docker
+import yaml
+
+app = FastAPI()
+client = docker.from_env()
+
+with open("config/monitored_services.yml", "r") as f:
+    config = yaml.safe_load(f)
+
+services_map = {entry["name"]: entry["container"] for entry in config["core_services"]}
```

Comment:
same add error handler when fail to load the configuration, file not found for example